README.md

```diff
@@ -39,7 +39,54 @@ brew install opus
 
 ## Usage example
 
-TODO
+### Encoder 
+Example encoder usage: 
+```elixir
+elements = [
```

Comment:
without the pipeline module, these examples are incomplete and not really helpful